lib/pbench/cli/agent/commands/tools/list.py

```diff
@@ -36,35 +45,45 @@ def execute(self):
             host_tools = {}
             for group in groups:
                 host_tools[group] = {}
-                for path in self.gen_tools_group_dir(group).glob("*/**"):
-                    if self.context.with_option:
-                        host_tools[group][path.name] = [
-                            p.read_text() for p in self.tools(path)
-                        ]
-                    else:
-                        host_tools[group][path.name] = [p for p in self.tools(path)]
+                try:
+                    for path in self.gen_tools_group_dir(group).glob("*/**"):
+                        if self.context.with_option:
+                            host_tools[group][path.name] = [
+                                p.read_text() for p in self.tools(path)
+                            ]
+                        else:
+                            host_tools[group][path.name] = [p.name for p in self.tools(path)]
+                except BadToolGroup:
+                    self.logger.error("Bad tool group name: %s", group)
```

Comment:
This should probably be the same message as line 70 below.

Reply:
Yes, it should. I'll fix it.